include/sys/emmc.h

```diff
@@ -0,0 +1,303 @@
+#ifndef _SYS_EMMC_H_
+#define _SYS_EMMC_H_
+
+#include <inttypes.h>
+#include <sys/mimiker.h>
+#include <sys/device.h>
+#include <sys/endian.h>
+#include <stddef.h>
+
+/* These operation modes are described in JESD86-A441, with a brief summary
+   at page 27 */
+typedef enum emmc_op_mode {
```

Comment:
Same comment like to line 32., about with wy is it "op" not "operation"

Reply:
Same, as before. Shorter names, are preferred as long as they are reasonable.